index.js

```diff
@@ -0,0 +1,50 @@
+window.onload = function () {
+  document.getElementById("total").addEventListener('change', handleChange);
```

Comment:
Your `handleChange` will fire only when the element loses focus. Some people don't know this and so nothing would happen immediately after they typed in the second input field. Would be nice to either have a button for explicitly triggering the calculation, or change the second event from `change` to `keyup`.

Reply:
Great catch, thanks for this! I was trying to figure out how to get it to cause a change without a submit button. `keyup` sounds like a perfect option!